src/vision/Vision.cpp

```diff
@@ -122,8 +122,7 @@ int main(int argc, char **argv)
     ros::Subscriber rightCamSub = n.subscribe("/camera/left/image", 1, rightCamCallback);
```

Comment:
Can you remove the leading '/' on the topic? This makes the topic global, instead of relative.